issue_processing/jira_issue_processing.py

```diff
@@ -0,0 +1,401 @@
+        person = idservice.getPersonFromDB(idx)
+        user["email"] = person["email1"]  # column 'email1'
+        user["name"] = person["name"]  # column 'name'
+        user["id"] = person["id"]  # column 'id'
+
+        # add user information to buffer
+        # user_string = get_user_string(user["name"], user["email"]) # update for
+        buffer_db[user_string] = user
+
+        return user
+
+    for issue in issues:
+        # check database for issue author
+        issue["user"] = get_or_update_user(issue["user"])
+
+        # check database for event authors
+        for comment in issue["comments"]:
+            # get the event user from the DB
+            comment["author"] = get_or_update_user(comment["author"])
+            ## get the reference-target user from the DB if needed
+            #if event["ref_target"] != "":
+             #   event["ref_target"] = get_or_update_user(event["ref_target"])
+
+    log.debug("number of issues after insert_user_data: '{}'".format(len(issues)))
+    return issues
+
+
+def print_to_disk(issues, results_folder):
+    """Print issues to file 'issues.list' in result folder
+
+    :param issues: the issues to dump
+    :param results_folder: the folder where to place 'issues.list' output file
+    """
+
+    # construct path to output file
+    output_file = os.path.join(results_folder, "jira-issues.list")
+    log.info("Dumping output in file '{}'...".format(output_file))
+
+    # construct lines of output
+    lines = []
+    for issue in issues:
+        log.info("Current issue '{}'".format(issue['externalId']))
+        lines.append((issue['user']['username'],
+                      issue['user']['email'],
+                      issue['externalId'],
+                      issue['creationDate'],
+                      issue['externalId'],
+                      issue['type']))
+        for comment in issue["comments"]:
+            lines.append((
+                comment['author']['username'],
+                comment['author']['email'],
+                comment["id"],
+                comment['changeDate'],
+                issue['externalId'],
+                "comment"
+            ))
+
+    # write to output file
+    csv_writer.write_to_csv(output_file, lines)
+
+def print_to_disk_extr(issues, results_folder):
+    """Print issues to file 'issues.list' in result folder
+
+    :param issues: the issues to dump
+    :param results_folder: the folder where to place 'issues.list' output file
+    """
+
+    # construct path to output file
+    output_file = os.path.join(results_folder, "issues.list")
+    log.info("Dumping output in file '{}'...".format(output_file))
+
+    # construct lines of output
+    lines = []
+    for issue in issues:
+        log.info("Current issue '{}'".format(issue['externalId']))
+        for comment in issue["comments"]:
+            lines.append((
+                issue['externalId'],
+                issue['state'],

```

Comment:
Just as a cross-reference here: I opened issue se-passau/codeface-extraction-r#91  on the statuses that can be used in Jira.